components/studentPortal/freemium/modals/Welcome.tsx

```diff
@@ -2,7 +2,25 @@ import React from "react";
 export default function Welcome() {
   return (
     <div className="md:space-y-10 space-y-2">
-      Welcome to Skillify Coding Academy!
+      <h1 className="md:text-3xl text-lg font-bold text-center mt-6 md:mt-0">
+        Welcome to Skillify Coding Academy!
+      </h1>
+      <div className="flex flex-col md:flex-row h-80 md:h-full items-center">
+        <div className="flex flex-col space-y-2 md:space-y-8 text-sm md:text-base bg-[#E2E3FB] p-6 rounded-lg md:mt-0 mt-4">
```

Comment:
are you sure we want to use this one off colour, instead of adding it to our theme?

Reply:
added a color called sky to the tailwind config file.